charts/policy-controller/values.yaml

```diff
@@ -84,6 +84,8 @@ commonTolerations: []
 
 ## This will set some annotations in config maps and secrets. Use case: Disable versioning to deploy helm chart using spinnaker
 commonAnnotations: {}
+#  strategy.spinnaker.io/versioned: "false"
```

Comment:
should these commented lines be removed?

Reply:
@k4leung4 It is just an example how to set the annotation. That's why it is commented out :)